persistence/src/main/scala/hmda/persistence/institutions/SubmissionPersistence.scala

```diff
@@ -29,16 +29,23 @@ object SubmissionPersistence {
       event match {
         case SubmissionCreated(s) =>
           SubmissionState(s +: submissions)
+
         case SubmissionStatusUpdated(id, status) =>
-          val x = submissions.find(x => x.id == id).getOrElse(Submission())
+          val x: Submission = submissions.find(x => x.id == id).getOrElse(Submission())
```

Comment:
Oops, that was probably left over from the debugging process. Fixed in a5804cd6.